VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.tsx

```diff
@@ -39,13 +38,7 @@ const IncorrectServiceInfo: FC<IncorrectServiceInfoScreenProps> = () => {
         <TextView accessibilityLabel={t('militaryInformation.incorrectServiceInfo.bodyA11yLabel.3')} variant="MobileBody" paragraphSpacing={true}>
           {t('militaryInformation.incorrectServiceInfo.body.3')}
         </TextView>
-        <ClickForActionLink
-          testID="incorrectServiceDMDCNumberTestID"
-          displayedText={displayedTextPhoneNumber(t('8005389552'))}
-          a11yLabel={a11yLabelID(t('8005389552'))}
-          numberOrUrlLink={t('8005389552')}
-          linkType={LinkTypeOptionsConstants.call}
-        />
+        <ClickToCallPhoneNumber phone={t('8005389552')} displayedText={displayedTextPhoneNumber(t('8005389552'))} />
```

Comment:
Don't we still need the `testID="incorrectServiceDMDCNumberTestID"`? Looks like that's referenced in the MilitaryInformation e2e test

Reply:
updated the e2e test to have the correct test ID